sdk/monitor/azure-monitor-query/README.md

```diff
@@ -234,6 +253,67 @@ for metric in response.metrics:
             print(metric_value.time_stamp)
 ```
 
+### Handling the response for metrics
+
+The metrics query API returns a `MetricsResult` object. This object has a list of `Metric` type along with other params like `interval`, `namespace` and `timespan`.
+
+This cab ne fetched using the `metrics` param.
+
+Each of the item in this list is a `Metric` object that contains a list of `TimeSeriesElement`. Each of the timeseries elements contain `data` and `metadata_values` params.
```

Comment:
nit - for readability + consistency, maybe something like this?
```suggestion
Each of the item in this list is a `Metric` object that contains a list of `TimeSeriesElement` objects. Each `TimeSeriesElement` contains `data` and `metadata_values` params.
```

Also, not sure about the context here, but did you mean to use the word `params` or is it moreso `properties`/`attributes`?